cytotable/convert.py

```diff
@@ -183,13 +175,14 @@ def _prep_cast_column_data_types(
 
 
 @python_app
-def _get_table_chunk_offsets(
+def _get_table_keyset_pagination_sets(
```

Comment:
It seems that `page_key` needs to be orderable and from the definition of `pagesets` elsewhere I presume it also has to be an integer, but is it the case that it also has to be unique? If there are repeats, I imagine you'd end up with page sizes larger than the chunk size you specified. Perfectly fine if the page key needs to be unique, but that's something I'd add to the documentation if so.

You might also consider tightening the return type from `List[Any]` to `List[Tuple[int,int]]` to be more explicit about the chunks being a list of integer key pairs that define each page.

Reply:
Thanks good point! Yes, there are some assumptions we should document upfront for the `pagesets`. I tried to be loose in the considerations here but there are a few gotchas which resulted in a more complicated implementation of the `pageset` list of tuples implementation (will touch on this in the other comment).